pyproject.toml

```diff
@@ -34,13 +34,16 @@ boto3 = ">=1.0.0"
 pytz = ">=2016.10"
 django = ">=2.2"
 importlib-metadata = {version = "^1.0", python = "<3.8"}
+cryptography = {version = "^36.0.2", optional = true}
+requests = {version = "^2.27.1", optional = true}
 
 [tool.poetry.extras]
-bounce =  ["requests<3", "M2Crypto"]
-events = ["requests<3", "M2Crypto"]
```

Comment:
I assume you tested on `requests>=3`? Did you check anywhere else requests is used? I'm glad to get rid of this, just want to make sure we're not breaking anything.

Reply:
Requests isn't used anywhere else in django-ses. 

The version is still locked, as I understand it, by the optional requirement above, which reads:

```
requests = {version = "^2.27.1", optional = true}
```

I haven't found good documentation of how extras work with poetry though. 

What would you think about not doing extras anymore as part of this version bump and just making cryptography and requests required?

Reply:
If we ditched extras then cryptography would have to be installed to use django-ses. `cryptography` and `requests` would just be required dependencies.

> I'm fine with requiring the extras, but I'm also open to trying this since you already did it.

Just to be clear, I think the toml file right now still uses extras more or less as the package always has. All of this should work:

```
pip install django-ses
pip install django-ses[events]
pip install django-ses[bounce]
```

I'd suggest leaving it be, I think, if having cryptography as an option install is desired.